src/types/index.ts

```diff
@@ -247,6 +247,13 @@ export interface SelectProps {
      * @category Custom Component
      */
     OptionComponent?: (props: OptionComponentProps) => JSX.Element;
+    /**
+     *  If `true` toggling the select is animated
+     *
+     *  @category Styles
+     *  @default true
+     */
+    isAnimated?: boolean;
```

Comment:
This prop should be named `animation` instead `isAnimation`. Similar like other bool props like `clearable`, `searchable` etc.

Reply:
maybe animated?

Reply:
`animated` sounds good